lib/x11regressiontest.pm

```diff
@@ -481,25 +481,34 @@ sub exit_firefox() {
 }
 
 sub start_gnome_settings {
-    my $is_sle_12_sp1 = (check_var('DISTRI', 'sle') && check_var('VERSION', '12-SP1'));
-    my $workaround_repetitions = 3;
+    my $is_sle_12_sp1          = (check_var('DISTRI', 'sle') && check_var('VERSION', '12-SP1'));
+    my $workaround_repetitions = 5;
+    my $i                      = $workaround_repetitions;
+    my $settings_menu_loaded   = 0;
 
     # the loop is a workaround for SP1: bug in launcher. Sometimes it doesn't react to click
     # The bug will be NOT fixed for SP1.
     do {
         if ($is_sle_12_sp1) {
+            if ($i < $workaround_repetitions) {
+                record_soft_failure 'bsc#1041175 - The settings menu fails sporadically on SP1';
```

Comment:
Now I see what you are trying to do. If we know the bug will never be fixed then there should also not be a softfail. A softfail is a reminder that an open bug ist still present. A softfail that will never vanish is therefore also not helpful.